templates/swagger/v1_json.tmpl

```diff
@@ -16555,6 +16563,11 @@
           "format": "int64",
           "x-go-name": "StarredRepos"
         },
+        "visibility": {
+          "description": "User visibility level option: public (default), limited, private",
```

Comment:
Is public actually the default? If defatul is dynamic (my understanding) I'd only put
```suggestion
          "description": "User visibility level option: public, limited, private",
```

Reply:
Probably yes, do not need to mention it as default.